CONTRIBUTING.md

```diff
@@ -35,4 +34,80 @@ client_id="kafka-mgmt/v1"
 
 ```shell
 ./scripts/generate_go.sh $client_id $remote_openapi_url
-```
\ No newline at end of file
+```
+
+## Adding an API
```

Comment:
Those parts are more or less internal. 
My take is that upstream repo should be just having those files updated somehow but it is not for general knowledge where those come from etc.